src/components/Author/index.tsx

```diff
@@ -45,6 +46,7 @@ function Author({ image, username, date, favorites, isFavorited, slug }: Props)
           <div>
             <span className={styles.username}>{username}</span>
             <span>{date}</span>
+            <span data-testId="isUserFollow">{following ? '*** You follow this user' : ''}</span>
```

Comment:
The sentence if could be simpler. For example: 

```suggestion
            <span data-testId="isUserFollow">{following && '*** You follow this user'}</span>
```
You can use i18n for the texts